src/components/searchbar/SearchBar.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from "react";
+import './SearchBar.scss';
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import SearchIcon from '@material-ui/icons/Search';
+
+export default function ColorTextFields() {
+  return (
+    <form noValidate autoComplete="off">
```

Comment:
what is noValidate for? It might be needed but if it's not, then we should get rid of it